### PR TITLE
Fix setting `channelSetting` with SetInteriorVehicleData

### LINKED ISSUE #309: HMI renews Channel Set for current shown id only after switching screen to another item
### Bug Report
HMI renews `Channel Set` for current shown id only after switching screen to another item and back

##### Reproduction Steps
1. SDL and HMI are started
2. `REMOTE_CONTROL` app is registered and activated
3. Open `Audio` pop-up
4. Current shown channel is 1
5. Mobile app requests `SetInteriorVehicleData`(update `equalizerSettings` for channel id=1)

##### Expected Behavior
HMI update `Channel Set` for id 1 on `Audio` pop-up after receiving `RC.SetInteriorVehicleData` request

##### Observed Behavior
HMI renews `Channel Set` for id 1 on `Audio` pop-up only after switching current `equalizerSettings` item to another item and back

##### OS & Version Information
* sdl_hmi branch: [develop](https://github.com/smartdevicelink/sdl_hmi/tree/develop)
* commit: e8baa2d8f3f4b43d261323e0dd2c30843509e925
* sdl_core branch: https://github.com/smartdevicelink/sdl_core/tree/6.1.0_RC

### PR DESCRIPTION
**Pull request by YarikMamykin**
Implements/Fixes #309 

This PR is **ready** for review.

### Testing Plan
Manual testing

### Summary
In case of incoming update for equalizer data there was not correct update for temporary data on audio popup window.
So in this PR update of temporary values for audio popup window was reworked.

### CLA
- [X] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: app/model/media/AudioModel.js [hunk 1]
```diff
@@ -888,12 +888,12 @@ SDL.AudioModel = Em.Object.extend({
           resultNew.volume = data.volume;
         }
         for (var i = 0; i < dataLength; i++) {
+          if (data.equalizerSettings[i].channelId == this.tempEqualizerSettings.channelId) {
+            this.set('tempEqualizerSettings.channelSetting', data.equalizerSettings[i].channelSetting);
+          }
           for (var j = 0; j < currentLength; j++) {
             if (data.equalizerSettings[i].channelId == this.lastRadioControlStruct.equalizerSettings[j].channelId) {
               this.lastRadioControlStruct.equalizerSettings[j].channelSetting = SDL.deepCopy(data.equalizerSettings[i].channelSetting);
```
Comment (mkorniichuk) [inline app/model/media/AudioModel.js:896]: @YarikMamykin isn't this a good place for a break?

Comment (YarikMamykin) [reply app/model/media/AudioModel.js:896]: @mkorniichuk nope, because lastRadioControlStruct should also be updated. See code below.

Comment (mkorniichuk) [reply app/model/media/AudioModel.js:896]: @YarikMamykin but there is no mode code in the inner loop...

Comment (YarikMamykin) [reply app/model/media/AudioModel.js:896]: @mkorniichuk added in https://github.com/smartdevicelink/sdl_hmi/pull/338/commits/6763350acda18d1aa2e50e863f01ea71cae3a399

**Comment by YarikMamykin** [comment]
@theresalech this PR is ready for Livio team review. Please notice that https://github.com/smartdevicelink/sdl_hmi/issues/309 is **not** regression issue.

**Review by iCollin — APPROVED** [review]

**Pull request merged**